service_container.rst

```diff
@@ -607,6 +607,23 @@ accessor methods for parameters::
     To learn more about compiling the container see
     :doc:`/components/dependency_injection/compilation`.
 
+It is also possible to deprecate a container parameter. This is particularly useful
+if you're writing your own package and want to warn developers to not use a parameter anymore
+because it will be removed in one of the next version of your package.
+Deprecating a container parameter can be done as follow::
+
+    $container->deprecateParameter(
+        'vendor.foo',
```

Comment:
Let's use a more real example and not foo if possible

Reply:
Is it better now? 🙂